prime-router/src/test/kotlin/DocumentationTests.kt

```diff
@@ -79,4 +79,99 @@ $documentation
         val actual = DocumentationFactory.getElementDocumentation(elemWithDocumentation)
         assertThat(actual).isEqualTo(expected)
     }
+    @Test
+    fun `test documentation for element with type CODE with Format $display`() {
+        val elemWithTypeCode = Element(
+            "a",
+            type = Element.Type.CODE,
+            csvFields = Element.csvFields("b", format = "\$display")
+        )
+        val expected = """
```

Comment:
Just something I thought.  Another way to do this is do a string contains to check only for the portion you need (e.g. **Format**: use value ...)